main.go

```diff
@@ -68,13 +68,22 @@ var (
 )
 
 func probeHandler(w http.ResponseWriter, r *http.Request, c *config.Config, logger log.Logger, rh *resultHistory) {
+	probeModuleUnknownGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_module_unknown",
+		Help: "Displays whether or not probe module is unknown",
+	})
+	registry := prometheus.NewRegistry()
```

Comment:
This isn't right. This could be a counter using the default registry, and defined at file level.

Reply:
Think this is done now, please review